### PR TITLE
Add Phoenix Framework

### PR DESCRIPTION
**Pull request by emmabukacek**
## Changes
- Add initial phoenix framework.

@blad @kaylah

@sethetter See any bloat I can just rip out?

### CODE REVIEW

**Comment by stran-rms** [comment]
getinsertpic.com

File: api/priv/static/js/app.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// for phoenix_html support, including form and button helpers
+// copy the following scripts into your javascript bundle:
+// * https://raw.githubusercontent.com/phoenixframework/phoenix_html/v2.3.0/priv/static/phoenix_html.js
```
Comment (khennegin-rms) [inline api/priv/static/js/app.js]: newline

**Pull request merged**